src/solver/infeasible-problem-analysis/report.cpp

```diff
@@ -114,14 +116,12 @@ void InfeasibleProblemReport::logSuspiciousConstraints()
     {
         Antares::logs.error() << "* Binding constraints,";
     }
-
-    Antares::logs.error() << "* Negative hurdle costs on lines with infinite capacity (rare).";
```

Comment:
Couldn't see any use for this log line : 
it's printed in all cases of infeasibility, even if here are no link involved (indeed, **hurdle cost**s are related to links).
Besides, it can be misleading for user, forced to wonder why he's got this message.